src/get_target_data.py

```diff
@@ -380,7 +405,19 @@ def write_target_data(
     oracle_output_path = oracle_output_path / "oracle.parquet"
 
     oracle = target_data[1]
-    oracle.collect().write_parquet(oracle_output_path)
+    oracle_arrow = oracle.collect().to_arrow()
+
+    oracle_schema = pa.schema(
+        [
+            ("location", pa.string()),
+            ("target_date", pa.date32()),
+            ("clade", pa.string()),
+            ("oracle_value", pa.int32()),
+            ("nowcast_date", pa.string()),
```

Comment:
should this be a `pa.date32()` data type?

Reply:
See above comment with a follow-up question about the data format here.